practice-app/website/api/event.py

```diff
@@ -76,5 +79,74 @@ def event():
         except exc.SQLAlchemyError as e:
             db.session.rollback()
             return "Try Later", 403
```

Comment:
this one also should be 503, I guess.

Reply:
This issue does not exist after merging the master into this branch, and solved after resolving the conflicts.